src/config/mod.rs

```diff
@@ -218,6 +219,27 @@ mod test {
         assert_eq!(config.was_set().verbose(), false);
     }
 
+    #[test]
+    fn test_print_docs_without_unstable() {
+        let mut output = vec![];
+        Config::print_docs(&mut output, false);
+
+        let s = str::from_utf8(&output).unwrap();
+
+        assert_eq!(s.contains("newline_style"), true);
```

Comment:
I don't like how brittle this makes the test with respect to the options defined in the `create_config!` macro above.  I'd love some guidance on the best way to test this change.

Reply:
You could make another usage of `create_config` in a different module with mock data and test using that, rather than the real one - `Config` is the only name you can use with that macro, but if you use it in a different module, then it is a different name. You could do this within the test module and then only refer to it using the qualified path, e.g., something like:

```
mod mock_config {
    create_config! { ... }
}

#[test]
fn test_...() {
    mock_config::Config::print_docs(...);
}
```